apps/storefront/src/theme.tsx

```diff
@@ -39,10 +40,10 @@ function B3ThemeProvider({ children }: Props) {
       {
         palette: {
           background: {
-            default: backgroundColor,
+            default: platform === 'catalyst' ? '#FFFFFF' : backgroundColor,
           },
           primary: {
-            main: primaryColor || '#1976d2',
+            main: platform === 'catalyst' ? '#000000' : primaryColor || '#1976d2',
```

Comment:
🤔 do we receive a color from the configuration if none is selected in the dashboard?

This change will lock the color to `black` and `white` when using catalyst, not sure if that's the best course of action here, as we'll be disabling color configuration from the dashboard.

Ideally we want to use these new defaults if no color was selected.